napari/_tests/test_magicgui.py

```diff
@@ -346,3 +347,21 @@ def test_layers_populate_immediately(make_napari_viewer):
     assert not len(labels_layer.choices)
     viewer.window.add_dock_widget(labels_layer)
     assert len(labels_layer.choices) == 1
+
+
+def test_from_layer_data_tuple(make_napari_viewer):
```

Comment:
Nit: consider renaming or modifying the comment to indicate this is specifically about testing a `_DeprecatingDict`.